### PR TITLE
Qualys - Fixed an issue where command vmScanFetch did not have outputs

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29656

## Description
Added outputs to command qualys-vm-scan-fetch

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by abaumgarten** [comment]
@evisochek please update new outputs format in the issue

File: Packs/qualys/Integrations/integration-qualys.yml [hunk 1]
```diff
@@ -433,7 +433,8 @@ script:
         'vmScanAction': { ContextPath: undefined,
                         ContentPath: ['SIMPLE_RETURN.RESPONSE','.ITEM_LIST.ITEM'],
                          entityType: ''},
-        'vmScanFetch': '',
+        'vmScanFetch': { ContextPath: 'Qualys.VM',
```
Comment (teizenman) [inline Packs/qualys/Integrations/integration-qualys.yml:436]: No DT is required here?

Comment (abaumgarten) [reply Packs/qualys/Integrations/integration-qualys.yml:436]: No, i have checked it. The output of that command is different from other commands.

Comment (teizenman) [reply Packs/qualys/Integrations/integration-qualys.yml:436]: And what happens when you run the same command more than once? Then the context would have duplicates or not?

Comment (abaumgarten) [reply Packs/qualys/Integrations/integration-qualys.yml:436]: No, It will override

Comment (abaumgarten) [reply Packs/qualys/Integrations/integration-qualys.yml]: Change to DT

**Review by teizenman — APPROVED** [review]

**Review by guykeller — APPROVED** [review]

**Review by teizenman — CHANGES_REQUESTED** [review]
Please update the context outputs as agreed

**Review by teizenman — APPROVED** [review]

**Comment by teizenman** [comment]
@kirbles19 
Can you please go over the following items:
1. new context outputs: https://github.com/demisto/content/blob/Qualys-fix/Packs/qualys/Integrations/integration-qualys.yml#L2431:L2443
2. RN

**Comment by ShirleyDenkberg** [comment]
@guykeller Doc review completed.

**Pull request merged**